cockatrice/src/user_context_menu.cpp

```diff
@@ -89,6 +94,27 @@ void UserContextMenu::banUser_processUserInfoResponse(const Response &r)
     dlg->show();
 }
 
+void UserContextMenu::adjustMod_processUserResponse(const Response &resp, const CommandContainer &commandContainer)
+{
+
+    const Command_AdjustMod &cmd = commandContainer.admin_command(0).GetExtension(Command_AdjustMod::ext);
+
+    if (resp.response_code() == Response::RespOk) {
+        if (cmd.should_be_mod()) {
+            QMessageBox::information(static_cast<QWidget *>(parent()), tr("Success"), tr("Successfully promoted user."));
+        } else {
+            QMessageBox::information(static_cast<QWidget *>(parent()), tr("Success"), tr("Successfully demoted user."));
+        }
+
+    } else {
+        if (cmd.should_be_mod()) {
+            QMessageBox::information(static_cast<QWidget *>(parent()), tr("Failed"), tr("Failed to promote user."));
+        } else {
+            QMessageBox::information(static_cast<QWidget *>(parent()), tr("Failed"), tr("Failed to demote user."));
```

Comment:
For all these messages, can you include the name of the person I promoted? If I do multiple in a row I won't be able to tell the difference if the server responds slowly.